README.md

```diff
@@ -52,6 +52,26 @@ git.run(:pull, 'origin', 'master', timeout: 2) # override default timeout of 10
 git.run(:status) # will raise an error because :status is not in list of allowed commands 
 ```
 
+Debugging and Logging
+---------
+If you need insight to what commands you're running you can pass CommandRunner an object responding to :puts
+such as $stdout, $stderr, the writing and of an IO.pipe, or a file opened for writing. CommandRunnerNG will
+log start, stop, end timeouts. Eg:
+
+```rb
+require 'command_runner'
+
+CommandRunner.run('ls /tmp', debug_log: $stdout)
```

Comment:
Thanks for that - I for one didn't actually know that `$stdout` was the thing that responded to `puts`, so I would have been researching that. 👍 🍺 🍻